Dockerfile

```diff
@@ -1,10 +1,16 @@
-FROM alpine:3.2
+FROM ruby:2.3-alpine
 
 MAINTAINER PRX <sysadmin@prx.org>
 
-RUN apk update && apk --update add ca-certificates ruby ruby-irb ruby-json ruby-rake \
-    ruby-bigdecimal ruby-io-console libstdc++ tzdata postgresql-client nodejs \
-    linux-headers libc-dev zlib libxml2 libxslt libffi less
+RUN apk update && apk --update add \
+    ca-certificates \
+    tzdata \
+    linux-headers \
+    libxml2 \
+    libxslt \
+    postgresql-client \
+    nodejs \
```

Comment:
it's for the asset pipeline, and we do actually compile assets for this one.
I could do some more work and dismantle the use of the pipeline in this app, so it would not complain it needs js support, but it adds little to the size, and means I can add pages as needed easily.

I had thought for awhile we might have some admin pages for this app, but I am increasingly thinking we will not, and interact with the API for whatever might need to be done, like using a slack action/bot to ping a feed or see the last time an episode dropped, so I am increasingly ok with dropping the asset pipeline and nodejs - what do you think @cavis?